Classes/Context/TopwireContextFactory.php

```diff
@@ -90,6 +86,10 @@ private function resolveRenderingPath(string $extensionName, string $pluginName,
      */
     private function resolveContextRecord(?string $contextRecordId, ?int $pageUid = null): ContextRecord
     {
+        $pageUid ??= $this->request->getAttribute('routing')?->getPageId();
```

Comment:
Should we use this instead?

```suggestion
        $pageUid ??= $this->request->getAttribute('frontend.page.information')?->getId();
```

Reply:
Just a feeling that it would sound better here to use the id from the current page instead of extracting it from the routing - but I really have no opinion on that, so we can keep it as it is.